### ISSUE TITLE
Find the changes need in the ll file generated on windows

### ISSUE THREAD

**Issue by manuranga** [ballerina-platform/nballerina#90]
**Description:**
Find the minimal modification needed in the generated `ll` files to make it compile-able with clang

**Suggested Labels:**

**Suggested Assignees:**

**Affected Product Version:**

**OS, DB, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

**Comment by KavinduZoysa** [comment]
I have added 2 .ll files to compare in [here](https://github.com/KavinduZoysa/llvm). Since generated .ll file in windows does not work, I took bellow steps. (Let's take .ll file generated in windows as **windows.ll** and in Linux **linux.ll**)

- In windows.ll file, method names are represented as, `0` and `1`. So I manually changed them as `main` and `test`. Then it starts to work in windows. Also, there was one instruction mismatch.
- I took the linux.ll file and changed the `target datalayout` and `target triple` in order to match with windows. Then linux.ll file starts to work in windows.
